GoogleTestAdapter/Core.Tests/GoogleTestDiscovererTests.cs

```diff
@@ -302,12 +302,12 @@ private void AssertIsGoogleTestExecutable(string executable, bool isGoogleTestEx
                 .Be(isGoogleTestExecutable);
         }
 
-        private void FindTests(string location)
+        private void FindTests(string location, int? expectedNrOfTestCases = null)
```

Comment:
Using a plain `int` parameter with default value `TestResources.NrOfTests` is slightly clearer imho

Reply:
Yes you are right.  I have tidied this up, it is much clearer and less code.